chart/k8gb/values.yaml

```
@@ -69,11 +69,7 @@ externaldns:
   # -- extra volume mounts
   extraVolumeMounts: []
   # -- external-dns image repo:tag
-  # It is important to use the image from k8gb external-dns fork to get the full
-  # functionality. See links below
-  # https://github.com/k8gb-io/external-dns
-  # https://github.com/k8gb-io/external-dns/pkgs/container/external-dns
-  image: ghcr.io/k8gb-io/external-dns:v0.13.4-azure-ns-multiarch
+  image: registry.k8s.io/external-dns/external-dns:v0.15.0
```

```suggestion
  image: registry.k8s.io/external-dns/external-dns:v0.15.1
```
As discussed in slack and the community meeting, version 0.15.1 contains our upstream changes